integration-cli/docker_cli_rm_test.go

```diff
@@ -9,34 +10,50 @@ import (
 )
 
 func (s *DockerSuite) TestRmContainerWithRemovedVolume(c *check.C) {
-	testRequires(c, DaemonIsLinux)
 	testRequires(c, SameHostDaemon)
 
-	dockerCmd(c, "run", "--name", "losemyvolumes", "-v", "/tmp/testing:/test", "busybox", "true")
+	prefix := ""
```

Comment:
I'm good with doing that in a separate PR, given that we already are using that code in several places